diff --git a/node_modules/o1js/dist/node/index.d.ts b/node_modules/o1js/dist/node/index.d.ts
index e3bc4de..a2159a1 100644
--- a/node_modules/o1js/dist/node/index.d.ts
+++ b/node_modules/o1js/dist/node/index.d.ts
@@ -4,6 +4,7 @@ export { Field, Bool, Group, Scalar } from './lib/core.js';
 export { createForeignField, ForeignField, AlmostForeignField, CanonicalForeignField, } from './lib/foreign-field.js';
 export { createForeignCurve, ForeignCurve } from './lib/foreign-curve.js';
 export { createEcdsa, EcdsaSignature } from './lib/foreign-ecdsa.js';
+export { EllipticCurve, Point, Ecdsa } from './lib/gadgets/elliptic-curve.js';
 export { Poseidon, TokenSymbol, ProvableHashable } from './lib/hash.js';
 export { Keccak } from './lib/keccak.js';
 export { Hash } from './lib/hashes-combined.js';
diff --git a/node_modules/o1js/dist/node/index.js b/node_modules/o1js/dist/node/index.js
index a3c6bd7..fec3825 100644
--- a/node_modules/o1js/dist/node/index.js
+++ b/node_modules/o1js/dist/node/index.js
@@ -3,6 +3,7 @@ export { Field, Bool, Group, Scalar } from './lib/core.js';
 export { createForeignField, } from './lib/foreign-field.js';
 export { createForeignCurve, ForeignCurve } from './lib/foreign-curve.js';
 export { createEcdsa, EcdsaSignature } from './lib/foreign-ecdsa.js';
+export { EllipticCurve, Point, Ecdsa } from './lib/gadgets/elliptic-curve.js';
 export { Poseidon, TokenSymbol } from './lib/hash.js';
 export { Keccak } from './lib/keccak.js';
 export { Hash } from './lib/hashes-combined.js';
diff --git a/node_modules/o1js/src/index.ts b/node_modules/o1js/src/index.ts
index 6445ab3..40e04f2 100644
--- a/node_modules/o1js/src/index.ts
+++ b/node_modules/o1js/src/index.ts
@@ -9,6 +9,7 @@ export {
 } from './lib/foreign-field.js';
 export { createForeignCurve, ForeignCurve } from './lib/foreign-curve.js';
 export { createEcdsa, EcdsaSignature } from './lib/foreign-ecdsa.js';
+export { EllipticCurve, Point, Ecdsa } from './lib/gadgets/elliptic-curve.js';
 export { Poseidon, TokenSymbol, ProvableHashable } from './lib/hash.js';
 export { Keccak } from './lib/keccak.js';
 export { Hash } from './lib/hashes-combined.js';
